bioblend/galaxy/histories/__init__.py

```diff
@@ -660,3 +660,29 @@ def download_history(self, history_id, jeha_id, outf,
         r.raise_for_status()
         for chunk in r.iter_content(chunk_size):
             outf.write(chunk)
+
+    def copy_dataset(self, history_id, dataset_id, hda_ldda='hda'):
+        """
+        Copy a dataset to a different history.
+
+        :type history_id: str
+        :param history_id: history ID to which the dataset should be copied
+
+        :type dataset_id: str
+        :param dataset_id: dataset ID
+
+        :type hda_ldda: str
+        :param hda_ldda: Source of the dataset to be copied: 'hda' (the default) or 'library'
+
+        :rtype: None
+        :return: None
+        """
+
+        payload = {
+            'content': dataset_id,
+            'source': hda_ldda,
+            'type': 'dataset'
+        }
+
+        url = self.gi._make_url(self, history_id, contents=True)
+        return self._post(payload=payload, url=url)
```

Comment:
```suggestion
        self._post(payload=payload, url=url)
```

Reply:
I think return it, otherwise we won't have any idea from the shell if it was successful

Reply:
Sure